.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+jobs:
+  build:
+    machine: true
+
+    steps:
+      - checkout
+
+      - run:
+          name: compose_to_be_removed
+          command: |
+            cd stand
+            pwd
+            ls
+            docker-compose up
+          background: true
+
+      - run:
+          name: install dockerize
+          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
```

Comment:
Could you please split that line?
Also dockerize should be present in circleci native images